lib/mixlib/shellout/windows/core_ext.rb

```diff
@@ -530,5 +568,39 @@ def format_creds_from_hash(hash)
       [ logon, passwd, domain ]
     end
 
+    # Retrieves the environment variables for the specified user.
+    #
+    def retrieve_environment_variables(token, env_list)
+      env_pointer = FFI::MemoryPointer.new(:pointer)
+      create_environment_block(env_pointer, token, false)
```

Comment:
```suggestion
      create_environment_block(env_pointer, token)
```

If this function is only ever called with a constant parameter, it can be hardcoded into the function rather than passed in every time.